UndertaleModTool/Windows/SettingsWindow.xaml

```diff
@@ -27,7 +27,7 @@
         <TextBlock Grid.Row="2" Grid.Column="0" Margin="3" Text="Game Maker: Studio 2 runtimes path" ToolTip="Required only if you want to run GMS2 games using the Studio runner rather than the .exe"/>
         <TextBox Grid.Row="2" Grid.Column="1" Margin="3" Text="{Binding GameMakerStudio2RuntimesPath}"/>
 
-        <TextBlock Grid.Row="3" Grid.Column="0" Margin="3" Text="Enable asset order swapping" ToolTip="Toggles dragging &amp; dropping assets in the asset tabs to a different position in the list. Disabled by default."/>
+        <TextBlock Grid.Row="3" Grid.Column="0" Margin="3" Text="Enable asset order swapping" ToolTip="Toggles dragging &amp; dropping assets in the asset tabs to a different positions in the list. Disabled by default."/>
```

Comment:
This does not correct the typo but rather makes the typo worse.

Reply:
I forget that I can't read. Should be fixed now.